src/plugins/action_generate_review_assignment_records/src/types.ts

```diff
@@ -5,6 +5,13 @@ export enum AssignmentStatus {
   SELF_ASSIGN = 'Available for self-assignment',
 }
 
+export interface ApplicationData {
+  applicationId: number
+  templateId: number
+  stageId: number
+  stageNumber: number
+}
```

Comment:
Do we need to fully define this type? Cos there's a lot more in it than just that. Just wondering, maybe it's best to leave it as `any` in the Actions if we're not going to fully define it, otherwise it could be misleading.

Reply:
Well, I was going to define, but when I tried adding ApplicationStatus and realise it doesn't have access to generated GraphQL types (as I saw there is a duplication of `AssignmentStatus`) I gave up, I think it would be best if we would be importing those types from there instead of duplicating otherwise if we end up changing this will be source of problems

Reply:
Except the Actions need to be "self-contained" and not import stuff from higher up in the application structure. Can we pass in the GraphQL types somehow?

Reply:
I think we need a why to use the GraphQL types on actions too - instead of recreating types which is dangerous.
Not really part of this PR though, but I suggest simple adding the same package and config to generate GraphQL types on each plugin